spec/corrector_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../corrector'
+
+describe Corrector do
+  context 'It should add a classroom' do
+    corrector = Corrector.new
+
+    it 'corrector should be an instance of class Corrector' do
+      expect(corrector).to be_an_instance_of(Corrector)
+    end
+
+    it 'should correct name with first letter in upper case' do
+      expect(corrector.correct_name('hamid')).to eq 'Hamid'
+    end
```

Comment:
- [ ] the `Corrector` is also responsible for making sure the name is not longer than 10 characters, please write a test for that as well.